### PR TITLE
Fix the install process and give clear instructions

### PR DESCRIPTION
**Pull request by cs-499243**
I have added a full step by step guide (with error correction) to installing myradio in the README file. While doing this there was an error that appeared after database credentials were added, requiring a slight change to dbschema.php (thanks stratford)

### CODE REVIEW

**Comment by URY-GitHub** [comment]
Thank you for this pull request, @cs-499243!

@colinroitt, would you mind reviewing this PR, please? Feel free to ask anyone for help if you're not quite sure what you're doing.

**Comment by ColinRoitt** [comment]
Although requested probably best if someone who's built myradio before looks over this so will leave to @mstratford

File: README.md [hunk 1]
```diff
@@ -18,25 +18,45 @@ If you have [Vagrant](https://www.vagrantup.com) installed and want to get
...
-- Input any first name and last name, an email (NOT a @york.ac.uk email) and a password
-- If you enter a @york.ac.uk email you will not be able to login at all
+Database Credentials:
+ - Hostname: localhost
+ - Database: myradio
+ - Username: myradio
+ - Password: myradio
+
+Other:
+ - To open up the virtual machine, find it in virtualbox and login with "vagrant", "vagrant"
```
Comment (mstratford) [inline README.md:58]: Looks good, also note you can type "vagrant ssh" in the console and get to the Vagrant's VM console that way 👍

File: README.md [hunk 2]
```diff
@@ -18,25 +18,45 @@ If you have [Vagrant](https://www.vagrantup.com) installed and want to get
...
+
+CONNECT:
+ - Open up "https://localhost:4443/myradio/" in a browser
+ - [Use Chrome as this often fails to run on Firefox]
+ - It will say "connection not private" so press "advanced" and then "proceed"
+
+DATABASE:
+ - On the intro screen press "Click here to continue"
+ - Enter the database details (see Database Credentials) and press Next
+ - Enter the database details AGAIN and press next
```
Comment (mstratford) [inline README.md:39]: Gah, i'll have a debug on this. I would love this not to be a thing.

File: README.md (hunk 1 above)
Comment (cs-499243) [reply README.md:58]: Yeah I only realised that after. Should I change it to something more general like "connect to the VM"?

Comment (mstratford) [reply README.md:58]: sure 👍

File: README.md (hunk 2 above)
Comment (cs-499243) [reply README.md:39]: It's not all that bad a problem, even if you aren't aware of it when going in. If it's not a rather simple fix I'm not sure there's all that much point

Comment (mstratford) [reply README.md:39]: Ok, i've now fixed this bug in 8372a4d, please remove this line 👍

**Pull request merged**